src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/TableRowProjection.java

```diff
@@ -16,15 +17,20 @@
 import io.stargate.sgv2.jsonapi.service.operation.query.SelectCQLClause;
 import io.stargate.sgv2.jsonapi.service.projection.TableProjectionDefinition;
 import java.util.HashMap;
+import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 /**
  * Projection used for Table Rows (as opposed to Collection Documents), built from command API
  * projection definitions (expressed in JSON).
  */
 public record TableRowProjection(
-    ObjectMapper objectMapper, TableSchemaObject table, List<ColumnMetadata> columns)
+    ObjectMapper objectMapper,
+    TableSchemaObject table,
+    List<ColumnMetadata> columns,
+    List<CqlIdentifier> orderByColumns)
```

Comment:
So TableRowProjection now has two parts.


List<ColumnMetadata> columns -> This is from TableProjectionDefinition, so what user wants in projection. 


List<CqlIdentifier> orderByColumns -> columns we have to read, even users do not want in projection, since we need in-memory sort.